junit/src/main/java/cucumber/api/junit/Cucumber.java

```diff
@@ -57,10 +62,10 @@ public Cucumber(Class clazz) throws InitializationError, IOException {
 
         ResourceLoader resourceLoader = new MultiLoader(classLoader);
         runtime = createRuntime(resourceLoader, classLoader, runtimeOptions);
-
+        formatter = runtimeOptions.formatter(classLoader);
```

Comment:
I don't understand this. Is this needed? This also happens in Runner#buildBackendWorlds which calls  runtimeOptions.getPlugins();